recipe/install.sh

```diff
@@ -0,0 +1,9 @@
+make install
+
+if [[ "$PKG_NAME" == *static ]]
+then
+	# relying on conda to dedup package
+	echo "Keeping all files, conda will dedupe"
```

Comment:
How do we know that `libnghttp2` will be the one getting the shared objects etc? Is it because it is built first?

Reply:
The static package should pin exactly the non static one . Then it will be installed into host and deduped by conda build